transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/internal/encoders/AndroidClientInfoEncoder.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.android.datatransport.cct.internal.encoders;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.android.datatransport.cct.internal.AndroidClientInfo;
+import com.google.firebase.encoders.EncodingException;
+import com.google.firebase.encoders.ObjectEncoder;
+import com.google.firebase.encoders.ObjectEncoderContext;
+import java.io.IOException;
+
+public final class AndroidClientInfoEncoder implements ObjectEncoder<AndroidClientInfo> {
+  @Override
+  public void encode(
+      @Nullable AndroidClientInfo obj, @NonNull ObjectEncoderContext objectEncoderContext)
+      throws EncodingException, IOException {
+    if (obj.getSdkVersion() != Integer.MIN_VALUE) {
+      objectEncoderContext.add("SdkVersion", obj.getSdkVersion());
```

Comment:
I think all keys must start with a lower-case letter, i.e. `sdkVersion`, here and throughout

Reply:
Done